webpack.config.js

```diff
@@ -0,0 +1,186 @@
+const FriendlyErrorsWebpackPlugin = require('friendly-errors-webpack-plugin')
+const ProgressBarPlugin = require('progress-bar-webpack-plugin')
+const WebpackNotifierPlugin = require('webpack-notifier')
+const fs = require('fs')
+const path = require('path')
+const webpack = require('webpack')
+const { NODE_ENV, PORT } = process.env
+const isDevelopment = NODE_ENV === 'development'
+const isStaging = NODE_ENV === 'staging'
+const isProduction = NODE_ENV === 'production'
+const isDeploy = isStaging || isProduction
+
+const config = {
+  devtool: 'cheap-module-source-map',
+  entry: {
+    webpack: [
+      'webpack-hot-middleware/client?reload=true',
+      './desktop/apps/webpack/client.js'
+    ],
+    ...getEntrypoints()
+  },
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'public/assets'),
+    publicPath: '/assets',
+    sourceMapFilename: '[file].map?[contenthash]'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.coffee$/,
+        exclude: /node_modules/,
+        loader: 'coffee-loader'
+      },
+      {
+        test: /\.(jade|pug)$/,
+        exclude: /node_modules/,
+        loader: 'pug-loader',
+        options: {
+          doctype: 'html',
+          root: __dirname
+        }
+      },
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: [
+          {
+            loader: 'babel-loader',
+            query: {
+              cacheDirectory: true,
+              env: {
+                development: {
+                  presets: ['react-hmre'],
+                  plugins: [
+                    ['react-transform', {
+                      transforms: [{
+                        transform: 'react-transform-hmr',
+                        imports: ['react'],
+                        locals: ['module']
+                      }]
+                    }]
+                  ]
+                }
+              }
+            }
+          }
+        ]
+      },
+      {
+        test: /\.json$/,
+        loader: 'json-loader'
+      }
+    ]
+  },
+  plugins: [
+    new FriendlyErrorsWebpackPlugin({
+      compilationSuccessInfo: {
+        messages: [`[Force] Listening on http://localhost:${PORT} \n`]
+      }
+    }),
+    new ProgressBarPlugin(),
+    new WebpackNotifierPlugin(),
+    new webpack.DefinePlugin({
+      'process.env': {
+        'NODE_ENV': JSON.stringify(NODE_ENV)
+      }
+    }),
+    new webpack.NoEmitOnErrorsPlugin(),
+    new webpack.ProvidePlugin({
+      '$': 'jquery',
+      'jQuery': 'jquery',
+      'window.jQuery': 'jquery',
+      'jade': 'jade/runtime.js'
+    }),
+    new webpack.optimize.CommonsChunkPlugin({
+      name: 'common',
+      minChunks: 10 // lower number for larger "common.js" bundle size
+    })
+  ],
+  resolve: {
+    alias: {
+      'jquery.ui.widget': 'blueimp-file-upload/js/vendor/jquery.ui.widget.js',
+      'waypoints': 'waypoints/lib/jquery.waypoints.js'
```

Comment:
@craigspaeth / @kanaabe - can you direct me to a URL where these two libraries are being used?

Reply:
It's used all over various old views with "on scroll" behavior. For instance, see [this about part of the about page](https://github.com/artsy/force/blob/master/desktop/apps/about/client/view.coffee#L55) where it controls navigation. I _did_ see an error in my local console when I peaked...

![image](https://user-images.githubusercontent.com/555859/34392743-257b5d3e-eb1c-11e7-942c-1a7ca65107ec.png)